librz/arch/asm.c

```diff
@@ -1861,6 +1861,15 @@ rz_asm_colorize_asm_str(RZ_BORROW RzStrBuf *asm_str, RZ_BORROW RzPrint *p, RZ_NU
 	return colored_asm;
 }
 
+/**
+ * \brief Free a RzAsmParseParam
+ *
+ * \param p The parameter struct.
+ */
+RZ_API void rz_asm_parse_param_free(RZ_OWN RZ_NULLABLE RzAsmParseParam *p) {
+	free(p);
+}
```

Comment:
i'm not a big fan of this. but if the others are ok, then i'm ok. @XVilka @thestr4ng3r

Reply:
Yeah, the only reason I kept it, was that we might extend the parse parameters. So the function is mostly there for future extensions. But we can also remove it or do the macro thing if the others agree with you.